pom.xml

```diff
@@ -66,7 +66,7 @@ under the License.
   </distributionManagement>
 
   <properties>
-    <javaVersion>7</javaVersion>
+    <javaVersion>8</javaVersion>
```

Comment:
That is not possible because the entire Maven 3 baseline is Java 7.

Reply:
3.x forever on Java 7 ? No plan to at least require java 8 which is 5 years old, while Java 7 has been EOL'ed more than 4 years ago ?

Anyway, if that's really the only objection, I'll work on a different PR, but I'd rather not rewrite the whole code if this ends up being rejected for whatever reason.

Reply:
Not for ever, neither an objection, but simply an important point. Java 7 isn't EOL'ed. The tree is alive, but you have to compile your own. The problem simply is that would not work with any previous Maven version where we say Java 7+.

Reply:
Ok, i rewrote the usage of streams so that Java 7 only is required.

Reply:
so ridiculous we are soon in 2020.....